src/pages/Main/PetSitterWrapper/PetSitterWrapper.js

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import PetSitterList from '../PetSitterList/PetSitterList';
+import SortButton from '../../../Components/Button/SortButton';
+import styled from 'styled-components';
+
+const initialSortBtn = [
+  { id: 1, text: '가까운순', isClicked: true },
+  { id: 2, text: '인기순', isClicked: false },
+  { id: 3, text: '가격순', isClicked: false },
+];
+
+const PetSitterWrapper = ({ data }) => {
+  const [sortBtn, setSortBtn] = useState(initialSortBtn);
+
+  const sortHandler = item => {
+    setSortBtn(prev =>
+      prev.map(prevItem =>
+        prevItem.id === item.id
+          ? { ...prevItem, isClicked: true }
+          : { ...prevItem, isClicked: false }
+      )
+    );
+  };
```

Comment:
위 리뷰 관련해서, 단순히 currentSort를 바꿔주는 역할만 하면 될 것 같습니다.
```suggestion
  const handleSort = id => {
    setCurrentSort(id)
  };
```